docs/features/pixi_global_manifest.md

```diff
@@ -0,0 +1,143 @@
+# Pixi Global Manifest
+
+### Motivation
+
+`pixi global` is currently limited to imperatively managing CLI packages.
+The next iteration of this feature should fulfill the following needs:
+
+- Shareable global environments.
+- Allow managing complex environments with multiple packages as dependencies
+- Flexible exposure of binaries
+
+## Design Considerations
+
+There are a few things we wanted to keep in mind in the design:
+
+1. **User-friendliness**: Pixi is a user focussed tool that goes beyond developers. The feature should have good error reporting and helpful documentation from the start.
+2. **Keep it simple**: The CLI should be all you need to interact with the global environments.
+3. **Unsurprising**: Simple commands should still behave similar to traditional package managers.
+4. **Human Readable**: Any file created by this feature should be human-readable and modifiable.
+
+## Manifest
+
+```toml title="pixi-global.toml"
+# `python_3_10` is the name of the environment
+[envs.python_3_10.dependencies.python]
+spec = "3.10.*"
+expose_binaries = {"python3.10"="python"} # `python` from `python_3_10` will be available as `python3.10`
+
+[envs.python.dependencies.python]
+spec = "3.11.*"
+expose_binaries = "auto" # This will expose python, python3 and python3.11
+
+[envs.python.dependencies.pip]
+spec = "*"
+```
+
+## CLI
+
+Install one or more packages `PACKAGE` into their own environments and expose their binaries
+
+```
+pixi global install [PACKAGE]...
+```
+
+Remove environments of the named packages `PACKAGE`
+```
+pixi global uninstall [PACKAGE]...
+```
+
+Resolve environment without previously defined specs
+```
+pixi global upgrade python
+```
+
+Add package `PACKAGE` to environment `ENV`.
+If environment `ENV` does not yet exist, then it will be created.
+```
+pixi global add --name ENV PACKAGE
+```
+
+Remove package `PACKAGE` from environment `ENV`.
+```
+pixi global remove --name ENV PACKAGE
+```
+
+Update the version of one package
+```
+pixi global update --name python python=3.12.*
+```
+
+Set for a specific package `PACKAGE` in environment `ENV` under which `MAPPING` binaries are exposed
+```
+pixi expose-bin --name ENV --package PACKAGE [MAPPING]...
```

Comment:
I think we need to explicitly write what is the `[MAPPING]` syntax

Reply:
True!